packages/Overlay/src/Overlay.js

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Transition } from "react-transition-group";
+import FocusTrap from "focus-trap-react";
+import LockBodyScroll from "@paprika/helpers/lib/components/LockBodyScroll";
+import Portal from "@paprika/helpers/lib/components/Portal";
+import tokens from "@paprika/tokens";
+import Wrapper from "./Wrapper";
+import Backdrop from "./Backdrop";
+
+const propTypes = {
+  className: PropTypes.string,
```

Comment:
shouldn't this be part of `{...moreProps}` part of the root node?

Reply:
we shouldn't have it as a prop because we can be assigned on `...moreProps`

Reply:
Ah, I see. Just forgot to drop className from propTypes when introduced `...moreProps` (previously `className` was directly passed to `Wrapper`).